llvm/lib/Target/AArch64/AArch64SelectionDAGInfo.cpp

```diff
@@ -10,9 +10,13 @@
 //
 //===----------------------------------------------------------------------===//
 
+#include "AArch64SelectionDAGInfo.h"
 #include "AArch64TargetMachine.h"
 #include "Utils/AArch64SMEAttributes.h"
 
+#define GET_SDNODE_DESC
+#include "AArch64GenSDNodeInfo.inc"
+
```

Comment:
We typically undefined the generation macros straight after the include to avoid accidental uses.

Reply:
Most, if not all, generated headers undefine macros, so I just did the same.